Testscripts/Linux/KDUMP-Execute.sh

```diff
@@ -120,13 +120,15 @@ Check_KDUMP()
 
 Configure_NMI()
 {
-    sysctl -w kernel.unknown_nmi_panic=1
-    if [ $? -ne 0 ]; then
-        LogErr "Failed to enable kernel to call panic when it receives a NMI."
-        SetTestStateAborted
-        exit 0
-    else
-        UpdateSummary "Success: enabling kernel to call panic when it receives a NMI."
+    if [ -e "/proc/sys/kernel/unknown_nmi_panic" ] ; then
+        sysctl -w kernel.unknown_nmi_panic=1
+        if [ $? -ne 0 ]; then
+            LogErr "Failed to enable kernel to call panic when it receives a NMI."
+            SetTestStateAborted
+            exit 0
+        else
+            UpdateSummary "Success: enabling kernel to call panic when it receives a NMI."
+        fi
```

Comment:
It looks good to check out nmi kernel panic parameter. It could be helpful to add "else" block of new if-statement. It helps to read the test log later.